src/Container/Container.php

```diff
@@ -36,76 +41,131 @@ class Container implements ContainerInterface
      * @param mixed  $value
      * @return Container
      */
-    public function set($key, $value)
+    public function set(string $key, $value): self
```

Comment:
```suggestion
    public function set($key, $value): self
```

Reply:
@PaulPylkas can you check this, I think we need to remove the type declaration here or not?